creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -274,15 +293,75 @@ class SnapbackSM {
   }
 
   /**
-   * Submit POA:UserReplicaSetFactory.updateReplicaSet() transaction to move user off unhealthy node
+   * Depending on the size of `unhealthyReplicas`:
+   * 1. Determine a new replica set
+   * 2. Write new replica set to URSM
+   * 3. Sync data to new replica set
    *
-   * @dev TODO use libs for CreatorNodeSelection logic + relayed chain call
-   * @dev One issue - since it doesn't actually rectify broken replica sets, this will log every time it is processed
-   */
-  async issueUpdateReplicaSetOp (userId, wallet, primary, secondary1, secondary2, unhealthyReplica) {
-    // await libs.userReplicaSetManagerClient._updateReplicaSet(userId, primary, [new secondaries], primary, [old secondaries])
+   * @param {number} userId user id to issue a reconfiguration for
+   * @param {string} wallet wallet address of user id
+   * @param {string} primary endpoint of the current primary node on replica set
+   * @param {string} secondary1 endpoint of the current first secondary node on replica set
+   * @param {string} secondary2 endpoint of the current second secondary node on replica set
+   * @param {string[]} unhealthyReplicas array of endpoints of current replica set nodes that are unhealthy
+   * @param {string[]} healthyPeers array of endpoints of current peer set nodes slice that are healthy
```

Comment:
this is not actually `healthyPeers` right, it's not the current healthy peerset, its every healthy registered node?

Reply:
in this code flow, it was actually the healthy nodes, excluding what was passed in as part of the blacklist. with the refactor, it will be every healthy possible node at the time of the query